tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/BrowserBased/BrowserAdditionalInformation/Supplier Edit BrowserAdditionalInformation.3601.feature

```diff
@@ -16,15 +16,15 @@ Background:
-        | Sln1     | An full online medicine system | Online medicine 1 | { "AdditionalInformation": "Some Info", "ClientApplicationTypes": ["browser-based"],"BrowsersSupported" : [ "IE8", "Opera" ], "MobileResponsive": false, "Plugins": null, "MinimumConnectionSpeed": null, "MinimumDesktopResolution": null, "HardwareRequirements": "New Hardware" } |
+        | Solution | SummaryDescription             | FullDescription   | ClientApplication                        |
+        | Sln1     | An full online medicine system | Online medicine 1 | { "AdditionalInformation": "Some Info" } |
     When a PUT request is made to update the browser-additional-information section for solution Sln1
         | AdditionalInformation |
         | New Additional Info   |
     Then a successful response is returned
     And SolutionDetail exist
-        | Solution | SummaryDescription             | FullDescription   | ClientApplication                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
-        | Sln1     | An full online medicine system | Online medicine 1 | { "ClientApplicationTypes": ["browser-based"],"BrowsersSupported" : [ "IE8", "Opera" ], "MobileResponsive": false, "Plugins": null, "MinimumConnectionSpeed": null, "MinimumDesktopResolution": null, "HardwareRequirements": "New Hardware", "NativeMobileHardwareRequirements": null, "NativeDesktopHardwareRequirements": null, "AdditionalInformation": "New Additional Info", "MobileFirstDesign": null, "NativeMobileFirstDesign": null, "MobileOperatingSystems": null, "MobileConnectionDetails": null, "MobileMemoryAndStorage": null, "MobileThirdParty": null, "NativeMobileAdditionalInformation": null, "NativeDesktopOperatingSystemsDescription": null, "NativeDesktopMinimumConnectionSpeed": null, "NativeDesktopThirdParty": null, "NativeDesktopMemoryAndStorage": null } |
+        | Solution | SummaryDescription             | FullDescription   | ClientApplication                                                                                          |
+        | Sln1     | An full online medicine system | Online medicine 1 | { "ClientApplicationTypes": [], "BrowsersSupported" : [], "AdditionalInformation": "New Additional Info" } |
```

Comment:
Should this have "browser-based" in the ClientApplicationTypes?

Reply:
It could do... I figured as it is possible to modify without it having browser-based set, the test was cool to do the same. Really there's a case to be made for failing the PUT if you modify it without browser-based being set, but currently it's an accurate representation of the functionality.